float8_experimental/float8_dynamic_utils.py

```diff
@@ -130,20 +143,38 @@ def unwrap(t):
         )
 
     def __tensor_flatten__(self):
-        return ["_tensor"], self._mm_config
+        if self._pre_computed_amax:
```

Comment:
use if-else to handle optional tensor `_pre_computed_amax` otherwise dynamo will complain about `None`